db/trie/mptrie/leafnode_test.go

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) 2020 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package mptrie
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/iotexproject/iotex-core/db/trie"
+	"github.com/iotexproject/iotex-core/db/trie/triepb"
+)
+
+func TestLeafNodeProto(t *testing.T) {
+	require := require.New(t)
+	lnode := &leafNode{
+		key:   []byte("iotex"),
+		value: []byte("chain"),
+	}
+	proto, err := lnode.proto(nil, true)
+	require.NoError(err)
+	nodepb, ok := proto.(*triepb.NodePb)
+	require.True(ok)
+	leaf, ok := nodepb.Node.(*triepb.NodePb_Leaf)
+	require.True(ok)
+	lnode1 := newLeafNodeFromProtoPb(leaf.Leaf, nil)
+	require.Equal(lnode.key, lnode1.key)
+	require.Equal(lnode.value, lnode1.value)
+}
+
+func TestLeafNodeDelete(t *testing.T) {
```

Comment:
test cases in `TestLeafNodeUpsert` could be used in `TestLeafNodeDelete`

Reply:
Fixed